core/api/src/main/java/edu/uci/ics/texera/api/field/DateField.java

```diff
@@ -78,6 +81,9 @@ public boolean equals(Object obj) {
 
     @Override
     public String toString() {
+        if (localDate == null) {
+            return "DateField [value= ]";
+        }
         return "DateField [value=" + localDate.toString() + "]";

```

Comment:
just replace `toString()` with `String.valueOf()`

Reply:
I replaced `toString()` with `String.valueOf() in this function of both files but I did not make this change in `getDateString()` function of this file because the frontend will display "null" in stead of empty string if `getDateString()` is also changed. Is that ok?

Reply:
yes